src/vitals.c

```diff
@@ -196,7 +196,7 @@ void mmk_init_vital_functions(plt_ctx ctx)
     INIT_VITAL_FUNC(malloc);
     INIT_VITAL_FUNC(realloc);
     INIT_VITAL_FUNC(free);
-#ifdef MMK_EXE_FMT_ELF
```

Comment:
Removing this ifdef doesn't cause problems with Linux? The `HAVE_MMAP` variable is only checked for Windows.

Reply:
If I'm not mistaken, `HAVE_MMAP` is populated everywhere but on Windows (see [here](https://github.com/ros2/Mimick/blob/fc69876d2074af28d953dede6d633ba6a02c058f/CMakeLists.txt#L145-L152)).

Reply:
You are right, disregard my comment.